SITCOMTN-076.tex

```diff
@@ -49,6 +53,325 @@
+  We propose guidelines for the Project team to promote the generation and dissemination of high-quality documentation on the scientific performance.
+
+  \item Section \ref{operations} concludes with brief comments regarding the transition to Operations.
+
+\end{itemize}
+
+\section{Early Science Program}
+\label{early_science}
+
+Rubin Observatory has planned an Early Science Program to enable high-impact science prior to the first annual data release of the LSST \citedsp{RTN-011}.
+Components of the Early Science Program include releasing science-grade commissioning data products via a series of Data Previews, ramping up of the transient alert stream during commissioning, implementing a program of incremental template generation to augment alert production in the early phases of the survey, and the first LSST Data Release, DR1, based on the first 6 months of data from the LSST.
+
+Figure \ref{timeline} shows an overview of the expected timeline for release of Early Science data products and supporting documentation for the technical and scientific performance of Rubin Observatory.
+
+\begin{figure}
+  \includegraphics[width=1.0\textwidth]{timeline.png}
+  \caption{Expected timeline for release of Early Science data products and supporting documentation for the technical and scientific performance of Rubin Observatory.}
+  \label{timeline}
+\end{figure}
+
+Information regarding the technical and scientific performance of Rubin Observatory will be made available to the science community via technotes (Section \ref{technotes}) and a set of Construction papers (Section \ref{publications}).
+
+\begin{warning}[Drafting in progress!]
+  %The timeline for the release of PVIs, and release of associated source catalogs, is being reviewed by the Project Change Control Board.
+  %Release of associated source catalogs with the PVIs is not yet in the baseline.
+  %The proposed timeline for the release of PVIs and release of associated source catalogs are in the Project review process and may be revised
+  %The proposed timeline for release of PVIs is in the Project review process and may be revised.
+  This document is still being drafted.
+  Release of associated source catalogs with the PVIs is not in the baseline.
+  The proposed timeline for the release of PVIs and release of associated source catalogs are in the Project review process. After that, this document will be revised accordingly.
+\end{warning}
+
+%To support early science by the community,
+Rubin Observatory seeks to maximize opportunities for early science with LSST data by the community.
+At the start of the LSST 10-year survey, if the processing pipelines are sufficiently stable, nightly Processed Visit Images (PVIs) and accompanying source catalogs will be released on a best-effort basis.
```

Comment:
If the paragraph survives Eric's comment, I'd remove "If the pipelines are sufficiently stable".  We don't say, "If the telescope tracks sufficiently well", and that sort of thing seems a larger risk to me!

Reply:
This section is being shortened given updates to RTN-011